src/cases/store/reducers/index.ts

```diff
@@ -2,15 +2,18 @@ import {ActionReducerMap, createFeatureSelector} from '@ngrx/store';
 
 import * as fromCaseCreate from './create-case.reducer';
 import * as fromCaseSearchFilters from './search-filter.reducer';
+import * as fromCaseListFilters from './case-list-filter.reducer';
 
 export interface State {
   caseCreate: fromCaseCreate.CasesState;
   caseSearchFilter: fromCaseSearchFilters.SearchState;
+  caseListFilter: fromCaseListFilters.CaseFilterState;
 }
 
 export const reducers: ActionReducerMap<State> = {
   caseCreate: fromCaseCreate.reducerCreateCase,
-  caseSearchFilter: fromCaseSearchFilters.reducer
+  caseSearchFilter: fromCaseSearchFilters.reducer,
+  caseListFilter: fromCaseListFilters.reducerCaseListFilter
```

Comment:
can u please rename to caseLIst and please rename caseSearch (i know is not your code)

Reply:
Done